### ISSUE TITLE
click gives warnings about unicode_literals

### ISSUE THREAD

**Issue by carlgeorge** [mkdocs/mkdocs#734]
I installed mkdocs from pypi into a python2 virtual environment.

```
(test)$ python -V
Python 2.7.10
(test)$ pip list
backports.ssl-match-hostname (3.4.0.2)
certifi (2015.9.6.2)
click (5.1)
Jinja2 (2.8)
livereload (2.4.0)
Markdown (2.6.2)
MarkupSafe (0.23)
mkdocs (0.14.0)
pip (7.1.2)
PyYAML (3.11)
setuptools (18.3.2)
six (1.9.0)
tornado (4.2.1)
wheel (0.26.0)
```

When running `mkdocs`, this warnings is printed multiple times.

```
/home/carl/tmp/test/lib/python2.7/site-packages/mkdocs/cli.py:86: Warning: Click detected the use of the unicode_literals __future__ import.  This is heavily discouraged because it can introduce subtle bugs in your code.  You should instead use explicit u"" literals for your unicode strings.  For more information see http://click.pocoo.org/python3/
  @common_options
```

**Comment by waylan** [comment]
This has already been addressed in #708 and will be available in the next release. In the meantime, you can ignore the warning. The issue which the warning addresses does not affect MkDocs. For more information see the discussion in #703.

**Comment by carlgeorge** [comment]
Sorry about that, I searched the issues for unicode_literals, but apparently I only searched open issues, not closed ones.  My mistake, thanks for the quick response!
